README.md

```diff
@@ -3,161 +3,100 @@
 [![Build Status](https://travis-ci.org/weldr/welder-web.svg?branch=master)](https://travis-ci.org/weldr/welder-web)
 [![codecov](https://codecov.io/gh/weldr/welder-web/branch/master/graph/badge.svg)](https://codecov.io/gh/weldr/welder-web)
 
-The web interface for Composer!
+**The web interface for Composer!**
 
+Cockpit Composer is a common web interface and running as a [Cockpit](https://cockpit-project.org/) plugin. Cockpit Composer joins together with [Lorax Composer](https://github.com/weldr/lorax/tree/lorax-composer) to generate custom images suitable for deploying systems, or as images ready to upload the cloud.
```

Comment:
"common web interface" does not really explain anything. How about explaining what it is first, and then how it  works? There are also some words missing. How about

"Composer generates custom images suitable for deploying systems or uploading to the cloud. It integrates into Cockpit as a frontend for Lorax  Composer."